src/components/molecules/Button/index.tsx

```diff
@@ -12,13 +21,25 @@ const Button = ({ children, primary, secondary, cartButton }: ButtonProps) => {
       'bg-black px-5 py-2 text-2xl text-gray-900 bg-opacity-20 hover:bg-opacity-30 lg:text-3xl lg:font-light':
         secondary,
     },
+    {
+      'w-full text-gray-900 bg-blue-400 bg-opacity-50 py-4 text-2xl hover:bg-opacity-40':
+        formButton,
+    },
     {
       'w-full bg-lime-700 py-3 text-2xl text-white hover:bg-opacity-80':
         cartButton,
     },
+    {
+      'w-full flex items-center justify-center bg-blue-400 bg-opacity-30 py-4 text-2xl text-gray-900 disabled cursor-not-allowed':
+        loadingButton,
+    },
```

Comment:
What I think of is that there shouldn't be specific buttons inside Molecule. It should be generic component that can be used in any scenarios. For example:

secondary is used to address forms
primary is somewhere else
loading is a state, not an type, it shouldn't override styles just disable the button (which actually can be handled by disabled props)
cartButton shouldn't differ from specified button types (e.g. primary, secondary) just should have icon props, etc. 

if you have any questions dm